app/build.gradle.kts

```diff
@@ -6,12 +6,12 @@ plugins {
 
 android {
     namespace = "hr.dice.filipbionda.tmdbpractice"
-    compileSdk = 34
+    compileSdk = 35
```

Comment:
Don't change this without checking the whole app

Reply:
Ok. One of my dependencies uses newest version (35) so there I was getting error while building gradle